src/main/java/org/apache/commons/compress/harmony/pack200/Pack200Adapter.java

```diff
@@ -41,11 +42,11 @@ public void addPropertyChangeListener(final PropertyChangeListener listener) {
      *
      * @param value Completion between 0..1.
      */
-    protected void completed(final double value) {
+    protected void completed(final double value) throws IOException {
```

Comment:
Hello @yakovsh 
Thank for your updates.
Note that if you add or remove exceptions to public or protected method signatures, you must update their Javadoc comments ;-)
TY